tm2py/config.py

```diff
@@ -976,6 +975,7 @@ class HighwayConfig(ConfigItem):
     interchange_nodes_file: str = Field()
     apply_msa_demand: bool = True
     reliability: bool = Field(default=True)
+    maz_drive_distance_threshold: float = Field()
```

Comment:
If we are adding a new paramemter to the model config, we should create a corresponding PR with actual config changes to `tm2py-utils`.